docs/releasenotes/3.2.0.rst

```diff
@@ -66,9 +66,28 @@ arguments or variables overwritten before first use::
         Log        ${var2}  # this variable is used
         ${var1}    Overwrite Variable  # this one is only overwritten
 
+New rule for not consistent variable naming (#496)
+--------------------------------------------------
+
+Added new rule W0323 ``inconsistent-variable-name`` which reports variables with inconsistent naming::
```

Comment:
```suggestion
Added new rule W0323 ``inconsistent-variable-name`` which reports variables with inconsistent naming.
Since Robot Framework ignore underscores and spaces in variable names, and also normalizes them to
lowercase, this code should report new tule violations::
```